src/Language/PureScript/AST/Declarations.hs

```diff
@@ -232,6 +232,10 @@ getModuleName (Module _ _ name _ _) = name
 getModuleSourceSpan :: Module -> SourceSpan
 getModuleSourceSpan (Module ss _ _ _ _) = ss
 
+-- | Return a module's declarations.
```

Comment:
Perhaps best not to do this right now, but I would quite like to use a record type for `Module`, and encourage using the record accessor functions instead of pattern matching on the `Module` constructor, so that any changes to the data type definition don't require so many other changes throughout the codebase.